docs: update SECURITY.md

joshuajames-smith opened the pull request:
## Description

Large documentation update to SECURITY.md which has absorbed and expanded upon the previous version to help improve the gliff.ai GitHub Community Health Files.

joshuajames-smith committed:
docs: update SECURITY.md

Large documentation update to SECURITY.md which has absorbed and expanded upon the previous version to help improve the gliff.ai GitHub Community Health Files.

joshuajames-smith committed:
fix: remove line indent in introduction

Add 1 space (total 2 spaces) after line break in introduction to stop line indent from occurring.

joshuajames-smith committed:
feat: add subject to mailto link

Add subject `[GitHub%20-%20Security%20Vulnerability]` to email mailto link.

joshuajames-smith committed:
style: line break and text edits in contact

Add line break in `Contact` section for style consistency across documentation. Edit text to match other documentation around reporting for consistency.

joshuajames-smith committed:
style: remove spacing

Remove double space styling for better whole text consistency.

joshuajames-smith committed:
feat: add links to gliff.ai website

Add links to any mention of `gliff.ai` in the introductions body text to offer link to website.

joshuajames-smith committed:
fix: change contact email

Change contact email.

ChasNelson1990 reviewed (commented):
see comment in the contributing guidelines about no longer using a template for reporting?

joshuajames-smith committed:
docs: remove issue template, replace with email

Relates #1 

Removing reference to process of creating an issue in place with reporting via email for privacy and security.

joshuajames-smith commented:
> see comment in the contributing guidelines about no longer using a template for reporting?

@ChasNelson1990 updated!
